.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -26,4 +26,4 @@ contact_links:
     about: Interesting in working at cLabs? Apply here!
```

Comment:
Not part of the PR but probably should be "Interested" ?

Reply:
Fixed in commit [334e116](https://github.com/celo-org/wallet/pull/298/commits/334e116b3816197be2d8099e360a1e87e6a1da27)